### PR TITLE
docs: add docs for the ganachecli module

### PR DESCRIPTION
**Pull request by CommanderStorm**
this adds docs for the ganachecli
Part of the #2 initative

### CODE REVIEW

**Review by DDtKey — APPROVED** [review]

**Comment by DDtKey** [comment]
@CommanderStorm JFYI: I noticed that you use the past tense in commit messages, but the common practice is to use the present tense. Commits are usually written in the imperative mood, such as providing a command or instruction.

File: src/trufflesuite_ganachecli/mod.rs [hunk 1]
```diff
@@ -1,19 +1,55 @@
...
+/// ```
+/// use testcontainers_modules::{testcontainers::runners::SyncRunner, trufflesuite_ganachecli};
+///
+/// let instance = trufflesuite_ganachecli::GanacheCli::default()
+///     .start()
+///     .unwrap();
+/// let url = format!(
+///     "http://{host_ip}:{host_port}",
+///     host_ip = instance.get_host().unwrap(),
+///     host_port = instance.get_host_port_ipv4(GANACHE_CLI_PORT).unwrap()
```
Comment (DDtKey) [inline src/trufflesuite_ganachecli/mod.rs:32]: the issue is this line, `GANACHE_CLI_PORT` isn't imported into doctest

File: src/trufflesuite_ganachecli/mod.rs [hunk 2]
```diff
@@ -1,19 +1,55 @@
...
+pub const GANACHE_CLI_PORT: ContainerPort = ContainerPort::Tcp(8545);
+
+/// # Module to work with the [`Ganache CLI`] inside of tests.
+///
+/// Starts an instance of Meilisearch.
+/// This module is based on the official [`trufflesuite/ganache-cli` docker image] documented in the [documentation].
+///
+/// # Example
+/// ```
+/// use testcontainers_modules::{testcontainers::runners::SyncRunner, trufflesuite_ganachecli};
```
Comment (CommanderStorm) [inline src/trufflesuite_ganachecli/mod.rs:24]:
```suggestion
/// use testcontainers_modules::{testcontainers::runners::SyncRunner, trufflesuite_ganachecli};
/// use testcontainers_modules::trufflesuite_ganachecli::GANACHE_CLI_PORT
```

File: src/trufflesuite_ganachecli/mod.rs [hunk 3]
```diff
@@ -1,19 +1,56 @@
...
+/// use testcontainers_modules::trufflesuite_ganachecli::GANACHE_CLI_PORT
```
Comment (CommanderStorm) [inline src/trufflesuite_ganachecli/mod.rs:25]: So far for being smart and not breaking out an IDE for this

```suggestion
/// use testcontainers_modules::trufflesuite_ganachecli::GANACHE_CLI_PORT;
```

File: src/trufflesuite_ganachecli/mod.rs [hunk 4]
```diff
@@ -1,19 +1,56 @@
...
+
+/// # Module to work with the [`Ganache CLI`] inside of tests.
+///
+/// Starts an instance of Meilisearch.
+/// This module is based on the official [`trufflesuite/ganache-cli` docker image] documented in the [documentation].
+///
+/// # Example
+/// ```
+/// use testcontainers_modules::{testcontainers::runners::SyncRunner, trufflesuite_ganachecli};
+/// use testcontainers_modules::trufflesuite_ganachecli::GANACHE_CLI_PORT;
```
Comment (CommanderStorm) [inline src/trufflesuite_ganachecli/mod.rs:25]: And obviously, rustfmt has a bone to pick with the last change 👀 

```suggestion
/// use testcontainers_modules::{
///     testcontainers::runners::SyncRunner, trufflesuite_ganachecli,
///     trufflesuite_ganachecli::GANACHE_CLI_PORT,
/// };
```

**Pull request merged**